lib/IDE/ArgumentCompletion.cpp

```diff
@@ -284,6 +284,15 @@ void ArgumentTypeCheckCompletionCallback::sawSolutionImpl(const Solution &S) {
     // and the code completion token doesn’t have a label, we have a case like
     // `Point(|)`. Suggest the entire function signature.
     IncludeSignature = true;
+  } else if (!ParentCall->getArgs()->empty() &&
+             ParentCall->getArgs()->getExpr(0) == CompletionExpr) {
+    if (auto ParentExpr = CS.getParentExpr(ParentCall);
+        ParentExpr && CS.getParentExpr(ParentCall)->getArgs()) {
```

Comment:
Does this handle multi-level nested cases like `foo(x: bar.baz(<HERE>`, and `foo(x: 1 + bar(<HERE>`? Adding test cases would be nice.

Also, do you mean:

```suggestion
    if (auto ParentExpr = CS.getParentExpr(ParentCall);
        ParentExpr->getArgs()) {
```
?

Reply:
I added test cases. It did work but I changed it to be a search through all parent expressions anyway. That seems more failure proof. Good idea 👍🏽